src/wallet/test/db_tests.cpp

```diff
@@ -72,6 +72,7 @@ static std::vector<std::unique_ptr<WalletDatabase>> TestDatabases(const fs::path
     return dbs;
 }
 
+
```

Comment:
In 6189a20491f71e3a6f499d2a510b38bb914e00d9 "test: Add tests for readonly database access"

Unnecessary newline